src/layouts/home/user-list-item/index.tsx

```diff
@@ -1,45 +1,45 @@
-import { Dispatch, SetStateAction } from 'react';
+import { formatDistanceToNow } from 'date-fns';
+import { useMemo } from 'react';
 
-import { ChatBriefViewResponse } from '@api/index';
-import { getDateDistance } from '@layouts/home/user-list-item/utils';
+import { ChatBriefViewResponse } from '@api/lib';
```

Comment:
`@api/index`
Also, a tip: use the `type` keyword syntax when importing types, it will bring a little benefit to Typescript